### DISCUSSION TITLE
Is it possible to get the total width in-use in a workspace?

### DISCUSSION THREAD

**Discussion by masroof-maindak** [YaLTeR/niri#696] [Q&A] [answered]
I'm attempting to make a workspace indicator akin to a horizontal minimap of the current workspace, and couldn't find a way to get the sum of all columns in an active workspace in Niri, as per [this](https://yalter.github.io/niri/niri_ipc/enum.Event.html) at least.

**Comment by YaLTeR** [comment]
Not atm but expansions to the IPC are planned.

**Reply by masroof-maindak** [reply]
Gotcha, thanks.

**Comment by airtonix** [comment]
Something we can put into a waybar / ags / eww / whatever widget that can be seen on mouse over / keyboard combo would be great.

**Comment by masroof-maindak** [comment, marked as answer]
Closing because of https://github.com/YaLTeR/niri/pull/1265.
